tools/generate-helpers/pg-table-bindings/index.go.tpl

```diff
@@ -50,17 +50,29 @@ func (b *indexerImpl) Add{{.TrimmedType}}(deployment *{{.Type}}) error {
 	return nil
 }
 
+{{if eq .TrimmedType "Policy" }}
+func (b *indexerImpl) Add{{trimSuffix "y" .TrimmedType}}ies(_ []*{{.Type}}) error {
```

Comment:
Can we just add logic that will do this automatically? So a function that checks if it ends with a y then will change the name to ies? etc?

Reply:
you mean add a new func to the template funcs?

Reply:
Expect a lot more changes - there a bunch of types that end in 'y' which had their plural naming messed up.